daprdocs/content/en/reference/api/workflow_api.md

```diff
@@ -235,7 +235,7 @@ The API call will provide a JSON response similar to this:
 
 Parameter | Description
 --------- | -----------
-`runtimeStatus` | The status of the workflow instance. Values include: `RUNNING`, `TERMINATED`, `PAUSED`  
+`runtimeStatus` | The status of the workflow instance. Values include: `"RUNNING"`, `"COMPLETED"`, `"CONTINUED_AS_NEW"`, `"FAILED"`, `"CANCELED"`, `"TERMINATED"`, `"PENDING"`, `"SUSPENDED"`  
```

Comment:
@philliphoff - I presume this is "CANCELLED" and not "CANCELED"?

Reply:
TIL from Grammarly:

> Canceled or cancelled is the past tense of the verb to cancel. Both spellings are correct; Americans favor canceled (one l), while cancelled (two l's) is preferred in British English and other dialects. Canceling/cancelling and canceler/canceller follow the same pattern.

So either way works for me haha

Reply:
We use "CANCELED" in the code: https://github.com/microsoft/durabletask-go/blob/119b361079c45e368f83b223888d56a436ac59b9/internal/protos/orchestrator_service.pb.go#L59